job-service/src/main/java/com/hpe/caf/services/job/api/DatabaseHelper.java

```diff
@@ -462,6 +462,55 @@ public void cancelJob(final String partitionId, String jobId) throws Exception {
         }
     }
 
+    /**
+     * Cancels the specified jobs
+     */
+    public int cancelJobs(final String partitionId, String jobIdStartsWith, final List<String> labels, final String filter)
```

Comment:
```suggestion
    public int cancelJobs(final String partitionId, final String jobIdStartsWith, final List<String> labels, final String filter)
```

Reply:
Fine, I see now that this pattern is used elsewhere in the existing code, I've made a suggestion below on how it could have been avoided.